sdk/costmanagement/azure-resourcemanager-costmanagement/src/main/java/com/azure/resourcemanager/costmanagement/implementation/QueriesClientImpl.java

```diff
@@ -64,6 +64,7 @@ Mono<Response<QueryResultInner>> usage(
             @HostParam("$host") String endpoint,
             @PathParam(value = "scope", encoded = true) String scope,
             @QueryParam("api-version") String apiVersion,
+            @QueryParam("$skiptoken") String skiptoken,
```

Comment:
I don't think costmanagement have the `$skiptoken` in their API spec https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json#L1022-L1038 ?

This should be done in their API spec.